pkg/asset/manifests/dns.go

```diff
@@ -81,6 +84,19 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 			fmt.Sprintf("kubernetes.io/cluster/%s", clusterID.InfraID): "owned",
 			"Name": fmt.Sprintf("%s-int", clusterID.InfraID),
 		}}
+	case azuretypes.Name:
+		dnsConfig, err := icazure.NewDNSConfig()
+		if err != nil {
+			return err
+		}
+
```

Comment:
since there are no errors after this, maybe compact this change to directly setting `PublicZone` and `PrivateZone`

Reply:
fixed